src/config.c

```diff
@@ -2165,6 +2159,14 @@ static int numericParseString(typeData data, sds value, const char **err, long l
             return 1;
     }
 
+    /* Attempt to parse as an octal number */
+    if (data.numeric.flags & OCTAL_CONFIG) {
+        char* eptr;
```

Comment:
```suggestion
        char *eptr;
```
Better follows our coding style.